### ISSUE TITLE
Improve fluid:telemetry:OdspDriver:TreesLatest_end telemety

### ISSUE THREAD

**Issue by jatgarg** [microsoft/FluidFramework#5016]
## Describe the bug
It would be great to get from SPO and record in our telemetry total snapshot size. I.e. include blob sizes that were excluded from this snapshot because they are too big or are images.
An issue has been opened at server side: https://onedrive.visualstudio.com/SPIN/_boards/board/t/Team%20Prague/Backlog%20items/?workitem=982134
Once this is done, log it in driver telemetry.

**Comment by vladsud** [comment]
@jatgarg, the item on SPO side is marked as "Removed". Can you please follow up on what are next steps here?

**Comment by jatgarg** [comment]
They are planning to change the wire format for the response. So this is now on halt for this and will be done as per the new format. Looking for new issue id.

**Comment by vladsud** [comment]
Got it! This issue is basically needs https://github.com/microsoft/FluidFramework/issues/4589 to be resolve first.

**Comment by jatgarg** [comment]
Workitem in ado - 486
